html/elements/link.json

```
@@ -1485,12 +1485,12 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤80"
+                "version_added": "≤15"
```

Implemented via https://github.com/WebKit/WebKit/commit/79e39b932a8130d3f0ae90f37b341f8068c3c880 in WebKit [535.1](https://github.com/WebKit/WebKit/blob/79e39b932a8130d3f0ae90f37b341f8068c3c880/Source/WebCore/Configurations/Version.xcconfig#L24-L26).

Tested locally in Chrome 14 and 15, and the collector test only passes in 15, not 14.

```suggestion
                "version_added": "15"
```